css/main.css

```diff
@@ -0,0 +1,165 @@
+.font-Roboto-Condensed { font-family: 'Roboto Condensed', sans-serif; }
+.font-Roboto { font-family: 'Roboto', sans-serif; }
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 { font-family: 'Roboto Condensed', sans-serif; font-weight: bold; }
+a { text-decoration: none; color: black; }
+a:hover { color: red; text-decoration: none; }
+
+.height-100 { min-height: 800px; }
+.navCollapsible-container { position: fixed; z-index: 2; width: 240px; top: 60px; right: 0; padding: 0 !important; }
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class name can be used instead. See [Position](https://getbootstrap.com/docs/4.1/utilities/position/) in bootstrap documentation.

#### Please, lets do this for all other occurrences. :blush: